src/packagemetadata.cpp

```diff
@@ -45,26 +56,8 @@ QString PackageMetadata::version()
     return m_version;
 }
 
-void PackageMetadata::isStarted()
-{
-    qDebug() << "zypper called";
-}
-
-void PackageMetadata::isFinished( int v )
+void PackageMetadata::isFinished()
```

Comment:
Instead of isFinished(), what do you think of renaming this method to something like processingFinished(), or sizeAndVersionObtained() or even processingDone() ?